supervision/detection/line_counter.py

```diff
@@ -197,6 +202,50 @@ def __init__(
         self.text_padding: int = text_padding
         self.custom_in_text: str = custom_in_text
         self.custom_out_text: str = custom_out_text
+        self.display_in_count: bool = display_in_count
+        self.display_out_count: bool = display_out_count
+
+    def _annotate_count(
+        self,
+        frame: np.ndarray,
+        line_counter: LineZone,
+        text: str,
+        count_pos: bool,
```

Comment:
I'd rename this argument to something more descriptive. For example, `is_in_count`?